packages/lit-dev-content/site/docs/ssr/server-api.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Lit SSR server API
+eleventyNavigation:
+  key: 🧪 Server API
+  parent: Server-side rendering
+  order: 2
+---
+
+<details class="pre-release">
+  <summary> 🧪 Pre-release software: not for production.</summary>
+  
+  Lit labs packages are published to get feedback from the wider community. This code shouldn't be used in production, but we encourage you to try it out and [file issues](https://github.com/lit/lit/issues/new/choose) for any bugs you find. For general feedback, please use the GitHub [discussion](https://github.com/lit/lit/discussions).
+
+For more information about the Lit labs process, see [Lib Labs](/docs/libraries/labs/)
+
+</details>
+
+Lit SSR provides two different ways of rendering custom elements server-side: rendering in the [global scope](#global-scope) or via [VM modules](#vm-module), which utilizes Node's [`vm.Module`](https://nodejs.org/api/vm.html#class-vmmodule) which enables running code within V8 Virtual Machine contexts. The two methods differ primarily in how the DOM shim is loaded.
```

Comment:
What about workers?

Reply:
We use workers for the eleventy plugin but don't provide anything for users to use workers from labs/ssr package yet. They'd have to manually run scripts in worker threads that utilize the render with global dom shim.